docs/installation.md

```diff
@@ -0,0 +1,74 @@
+# Install instructions
+
+## ADE based installation
+
+TODO: describe how the ADE image is structured and which features already includes (ROS 2traces
```

Comment:
@LanderU Could you add a brief description of the ADE image?

Reply:
Yes, let me review what you've done and I'll add the remaining stuff.